packages/website/api/og.tsx

```diff
@@ -0,0 +1,111 @@
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          background: "#f6f6f6",
+          width: "100%",
+          height: "100%",
+          justifyContent: "center",
+          alignItems: "center",
+        }}>
+        <svg
+          width="172"
+          height="172"
+          viewBox="0 0 88 88"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M86.2096 50.3913L71.0426 41.6346C70.0478 41.0603 69.7097 41.2555 69.7097 42.4042V47.0617C69.7097 48.3465 70.3951 49.5336 71.5077 50.1759L82.6274 56.5959C83.2532 56.9572 83.6418 57.6305 83.6419 58.3533V68.7914L65.8319 58.5088L65.8318 40.6944L65.8317 15.9917C65.8317 12.7173 64.0703 9.66666 61.235 8.02988L49.6276 1.32835C46.7922 -0.308372 43.2693 -0.3088 40.434 1.32835L28.8264 8.02988C25.9819 9.67216 24.2296 12.7072 24.2296 15.9918V32.6096C24.2296 33.7583 24.5677 33.9535 25.5625 33.3791L29.5961 31.0503C30.7086 30.408 31.394 29.2209 31.394 27.9361V15.9917C31.394 15.2691 31.7828 14.5959 32.4088 14.2345L41.4485 9.01536L41.4486 28.685L26.0208 37.5921L5.4032 49.4959C2.5675 51.1331 0.806213 54.1839 0.806396 57.4577V70.8608C0.80658 74.1346 2.56769 77.1858 5.4032 78.8227L17.0106 85.5243C19.8552 87.1667 23.3597 87.1667 26.2043 85.5244L39.8201 77.6632C40.8149 77.0889 40.8149 76.6985 39.8201 76.1242L35.7866 73.7954C34.674 73.1531 33.3032 73.1531 32.1906 73.7954L22.622 79.3198C21.9962 79.6811 21.2188 79.6811 20.5928 79.3197L11.553 74.1005L27.812 64.7134L43.2397 73.6206L65.4086 86.4198C68.2443 88.057 71.767 88.0569 74.6021 86.4198L86.2095 79.7183C89.0446 78.0812 90.8065 75.0304 90.8063 71.7564L90.8064 58.3532C90.8064 55.0687 89.0541 52.0336 86.2096 50.3913ZM71.0195 80.2154C70.3941 80.5765 69.6166 80.5766 68.9908 80.2152L46.8219 67.416L31.3942 58.5088V48.7795C31.3942 47.6309 31.0561 47.4357 30.0613 48.01L26.0278 50.3388C24.9152 50.9812 24.2298 52.1683 24.2298 53.453V58.5088L7.97083 67.8959L7.97095 57.4574C7.97095 56.7352 8.35962 56.0618 8.98541 55.7005L31.1543 42.9013L45.0308 34.8896L53.4567 39.7542C54.4514 40.3285 54.7896 40.1333 54.7896 38.9847V34.3271C54.7896 33.0423 54.1041 31.8552 52.9915 31.2129L48.613 28.6849L48.613 9.01536L57.653 14.2347C58.2784 14.5958 58.6673 15.2691 58.6673 15.9917V41.5902L58.6675 58.5089L50.2417 63.3735C49.2469 63.9479 49.2469 64.3382 50.2417 64.9126L54.2753 67.2414C55.3879 67.8837 56.7586 67.8837 57.8712 67.2413L62.2497 64.7134L80.0598 74.9961L71.0195 80.2154Z"
+            fill="url(#paint0_linear_1_47)"
+          />
+          <defs>
+            <linearGradient
+              id="paint0_linear_1_47"
+              x1="76.5099"
+              y1="11.1618"
+              x2="28.624"
+              y2="93.632"
+              gradientUnits="userSpaceOnUse">
+              <stop stop-color="#00EBE7" />
+              <stop offset="1" stop-color="#6923BA" />
+            </linearGradient>
+          </defs>
+        </svg>
+        <div
+          style={{
+            fontSize: 60,
```

Comment:
Can we use the "Gilroy" font that's also used on the website? https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation/og-image-examples#using-a-custom-font